code/modules/crafting/tailoring.dm

```diff
@@ -1,53 +1,60 @@
+#define ALLOWED_INGREDIENT_SUNGLASSES list( \
+	/obj/item/clothing/glasses/sunglasses, \
+	/obj/item/clothing/glasses/sunglasses/noir, \
+	/obj/item/clothing/glasses/sunglasses/yeah, \
+	/obj/item/clothing/glasses/sunglasses/big \
+)
+
 /datum/crafting_recipe/durathread_vest
 	name = "Durathread Vest"
 	result = list(/obj/item/clothing/suit/armor/vest/durathread)
 	reqs = list(/obj/item/stack/sheet/durathread = 5,
 				/obj/item/stack/sheet/leather = 4)
-	time = 50
+	time = 5 SECONDS
 	category = CAT_CLOTHING
 
 /datum/crafting_recipe/durathread_helmet
 	name = "Durathread Helmet"
 	result = list(/obj/item/clothing/head/helmet/durathread)
 	reqs = list(/obj/item/stack/sheet/durathread = 4,
 				/obj/item/stack/sheet/leather = 5)
-	time = 40
+	time = 4 SECONDS
 	category = CAT_CLOTHING
 
 /datum/crafting_recipe/durathread_jumpsuit
 	name = "Durathread Jumpsuit"
 	result = list(/obj/item/clothing/under/misc/durathread)
 	reqs = list(/obj/item/stack/sheet/durathread = 4)
-	time = 40
+	time = 4 SECONDS
 	category = CAT_CLOTHING
 
 /datum/crafting_recipe/durathread_beret
 	name = "Durathread Beret"
 	result = list(/obj/item/clothing/head/beret/durathread)
 	reqs = list(/obj/item/stack/sheet/durathread = 2)
-	time = 40
+	time = 4 SECONDS
 	category = CAT_CLOTHING
 
 /datum/crafting_recipe/durathread_beanie
 	name = "Durathread Beanie"
 	result = list(/obj/item/clothing/head/beanie/durathread)
 	reqs = list(/obj/item/stack/sheet/durathread = 2)
-	time = 40
+	time = 4 SECONDS
 	category = CAT_CLOTHING
 
 /datum/crafting_recipe/durathread_bandana
 	name = "Durathread Bandana"
 	result = list(/obj/item/clothing/mask/bandana/durathread)
 	reqs = list(/obj/item/stack/sheet/durathread = 1)
-	time = 25
+	time = 3 SECONDS
```

Comment:
2.5 seconds --> 3, 0.5 seconds --> 1, 4.5 seconds --> 5.
You changed the time on some recipes without mentioning on the description, if you wish to standardize/tweak the times to build then it would be better for another PR since this one is focused on glasses.